docs/notebooks/bald.pct.py

```diff
@@ -0,0 +1,144 @@
+# %%
```

Comment:
I assume docstrings will come :)
can you please use a more descriptive name for the file, not abbreviation

Reply:
Yes, I am working on it
Just thinking if it is neater to put this on the active learning notebook along with the predictive variance acquisition function, what do you think?

Reply:
this one is specific to classificaiton problems, so perhaps we can keep it separate, but you should definitely point one to each other